### PR TITLE
Gh 1222/add "guard" section in advanced safe settings

### PR DESCRIPTION
**Pull request by MouazAlzahabi**

### CODE REVIEW

File: Multisig/Data/Services/Safe Client Gateway Service/SCGModels.swift [hunk 1]
```diff
@@ -527,7 +527,20 @@ extension SCGModels {
         var implementation: AddressInfoExtended
         var modules: [AddressInfoExtended]?
         var fallbackHandler: AddressInfoExtended?
+        var guardInfo: AddressInfoExtended?
```
Comment (DmitryBespalov) [inline Multisig/Data/Services/Safe Client Gateway Service/SCGModels.swift]: Did you try using ```var `guard`:```?

File: Multisig/Cross-layer/Configuration/Configuration.swift [hunk 2]
```diff
@@ -49,6 +49,8 @@ struct AppConfiguration {
 
         @ConfigurationKey("FALLBACKHANDLER_URL")
         var fallbackHandlerURL: URL
+        @ConfigurationKey("GUARD_URL")
```
Comment (sche) [inline Multisig/Cross-layer/Configuration/Configuration.swift:53]: missing spacing

File: Multisig/Data/Services/Safe Client Gateway Service/SCGModels.swift (hunk 1 above)
Comment (sche) [reply Multisig/Data/Services/Safe Client Gateway Service/SCGModels.swift]: +1

File: Multisig/UI/Settings/SafeSettingsViewController/AdvancedSafeSettingsViewController.swift [hunk 3]
```diff
@@ -94,17 +101,37 @@ extension AdvancedSafeSettingsViewController {
...
+                                          imageUri: info.logoUri,
+                                          indexPath: indexPath)
             } else {
                 return tableView.basicCell(
                     name: "Not set", indexPath: indexPath, withDisclosure: false, canSelect: false)
             }
         case Section.FallbackHandler.fallbackHandlerHelpLink:
-            return fallbackHandlerHelpLinkCell(indexPath: indexPath)
+            return helpLinkCell(text: "What is a fallback handler and how does it relate to the Gnosis Safe",
+                                               url: App.configuration.help.fallbackHandlerURL,
```
Comment (sche) [inline Multisig/UI/Settings/SafeSettingsViewController/AdvancedSafeSettingsViewController.swift]: lines not aligned

File: Multisig/UI/Settings/SafeSettingsViewController/AdvancedSafeSettingsViewController.swift [hunk 4]
```diff
@@ -94,17 +101,37 @@ extension AdvancedSafeSettingsViewController {
...
+                                               indexPath: indexPath)
+        case Section.GuardInfo.guardInfo(let info):
+            return addressDetailsCell(address: info.address,
+                                      title: namingPolicy.name(info: info),
+                                      imageUri: info.logoUri,
+                                      indexPath: indexPath)
+        case Section.GuardInfo.guardInfoHelpLink:
+            return helpLinkCell(text: "What is a guard and how that is used",
+                                    url: App.configuration.help.guardURL,
```
Comment (sche) [inline Multisig/UI/Settings/SafeSettingsViewController/AdvancedSafeSettingsViewController.swift]: and here

File: Multisig/UI/Settings/SafeSettingsViewController/AdvancedSafeSettingsViewController.swift [hunk 5]
```diff
@@ -156,8 +185,10 @@ extension AdvancedSafeSettingsViewController {
         return cell
     }
 
-    private func fallbackHandlerHelpLinkCell(indexPath: IndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueCell(HelpLinkTableViewCell.self, for: indexPath)
+    private func helpLinkCell(text: String, url: URL, indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueCell(HelpLinkTableViewCell.self, for: indexPath) as HelpLinkTableViewCell
```
Comment (sche) [inline Multisig/UI/Settings/SafeSettingsViewController/AdvancedSafeSettingsViewController.swift]: do we need here `as HelpLinkTableViewCell`?

**Review by DmitryBespalov — APPROVED** [review]

**Pull request merged**